docs/pages/includes/enterprise/samlauthentication.mdx

```diff
@@ -1,39 +1,51 @@
-## Enable default SAML authentication
 
-Configure Teleport to use SAML authentication as the default instead of the local
+- Configure Teleport to use SAML authentication as the default instead of the local
 user database.
 
-<ScopedBlock scope={["oss", "enterprise"]}>
+  <ScopedBlock scope={["oss", "enterprise"]}>
+  You can either edit your Teleport configuration file or create a dynamic
+  resource.
+  </ScopedBlock>
 
-You can either edit your Teleport configuration file or create a dynamic
-resource.
+  <Tabs>
+  <TabItem label="Static Config (Self-Hosted)" scope={["oss", "enterprise"]}>
 
-</ScopedBlock> 
+  Update `/etc/teleport.yaml` in the `auth_service` section and restart the `teleport` daemon.
 
-<Tabs>
-  <TabItem label="Static Config (Self-Hosted)" scope={["oss", "enterprise"]}>
- Update `/etc/teleport.yaml` in the `auth_service` section and restart the `teleport` daemon.
   ```yaml
   auth_service:
     authentication:
       type: saml
   ```
+
   </TabItem>
   <TabItem scope={["cloud"]} label="Dynamic Resources (All Editions)">
-  Create a file called `cap.yaml`:
+
+  Use `tctl` to edit the `cluster_auth_preference` value:
+
+  ```code
+  $ tctl edit cluster_auth_preference
```

Comment:
Note that `tctl edit` is only available in v12+, so you'll have to change this for backports to 11 and 10.